lifehack-pet/scripts/content.js

```diff
@@ -1,3 +1,35 @@
+// Listen for messages from the popup script
+chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
+  console.log(
+    sender.tab
+      ? "from a content script:" + sender.tab.url
+      : "from the extension"
+  );
+  if (request.message === "updatePet") {
+    console.log(`Message: ${request.message}`);
+    updatePet();
+    sendResponse({ status: "Image update success" });
+  }
+});
+
+function updatePet() {
```

Comment:
Very nice documentation skills Allard!

Reply:
Thank you @erwinqxy, please let me know where I can improve!